core/modules/qdisp/QdispPool.h

```diff
@@ -135,6 +135,25 @@ class QdispPool {
 public:
     typedef std::shared_ptr<QdispPool> Ptr;
 
+    /// poolSize - total number of threads in the pool
+    /// largestPriority - highest priority is 0, lowest possible priority is
+    ///            100 and is reserved for default priority. largestPriority=4 would
```

Comment:
Where this "magic" limit of 100 is enforced? And why it's not loaded from a configuration file?

Reply:
It's been changed to a member of the class and is used limit the value of largestPriority (set from configuration file). If someone needs more than 100 priorities, it's probably worth altering this class for that kind of use.